docs/components/bottom_sheets.md

```diff
@@ -0,0 +1,286 @@
+<!--docs:
+title: "Material sheets: bottom"
+layout: detail
+section: components
+excerpt: "Bottom sheets are surfaces containing supplementary content that are anchored to the bottom of the screen."
+iconId:
+path: /catalog/material-sheets-bottom/
+-->
+
+# Sheets: bottom
+
+[Bottom sheets](https://material.io/components/sheets-bottom) are surfaces containing supplementary content that are anchored to the bottom of the screen.
+
+![Example bottom sheet: modal bottom sheet](assets/bottom_sheets/SheetsBottom_hero.png)
+
+**Contents**
+
+- [Using bottom sheets](#using-bottom-sheets)
+- [Making bottom sheets accessible](#making-bottom-sheets-accessible)
+- [Standard bottom sheet](#standard-bottom-sheet)
+- [Modal bottom sheet](#modal-bottom-sheet)
+- [Expanding bottom sheet](#expanding-bottom-sheet)
+- [Theming bottom sheets](#theming-bottom-sheets)
+
+## Using bottom sheets
+
+Bottom sheets are supplementary surfaces primarily used on mobile.
+
+Before you can use bottom sheets, you need to import the Material Components package for Flutter:
+
+```dart
+package:flutter/material.dart
+```
+
+You need to be using a [`MaterialApp`](https://api.flutter.dev/flutter/material/MaterialApp-class.html).
+
+For more information on getting started with the Material for Flutter, go to the Flutter [Material library](https://api.flutter.dev/flutter/material/material-library.html) page.
+
+### Making bottom sheets accessible
+
+Flutter's APIs support accessibility setting for large fonts, screen readers, and sufficient contrast. For more information, go to Flutter's [accessibility](https://flutter.dev/docs/development/accessibility-and-localization/accessibility) and [internationalization](https://flutter.dev/docs/development/accessibility-and-localization/internationalization) pages.
+
+For more guidance on writing labels, go to [our page on how to write a good accessibility label](https://material.io/design/usability/accessibility.html#writing).
+
+## Types
+
+There are three types suitable for different use cases:
+1. [Standard bottom sheets](#standard-bottom-sheet) display content that complements the screen’s primary content and remain visible while users interact with the primary content.
+1. [Modal bottom sheets](#modal-bottom-sheet) are an alternative to inline menus or simple dialogs on mobile and provide room for additional items, longer descriptions, and iconography, and must be dismissed in order to interact with the underlying content.
+1. [Expanding bottom sheets](#expanding-bottom-sheet) provide a small, collapsed surface that can be expanded by the user to access a key feature or task to offer the persistent access of a standard sheet with the space and focus of a modal sheet.
```

Comment:
The descriptive content for each type can move to the type's section. The remaining list/links to type sections should be an inline list:

"There are three types of bottom sheets: 1\. standard... 2\. ... 3\. ..."